plugins/backend-ekg/src/Cardano/BM/Backend/Prometheus.lhs

```diff
@@ -94,17 +98,28 @@ spawnPrometheus ekg host port prometheusOutput = Async.async $
         [ case sv of
             Counter c -> renderNamedValue sk (int64Dec c)
             Gauge g -> renderNamedValue sk (int64Dec g)
-            Label l -> if isFloat l
-                         then renderNamedValue sk (byteString $ encodeUtf8 l)
-                         else mempty
+            Label l -> trace ("renderSamples " <> T.unpack l) $
```

Comment:
Please remove debug code

Reply:
Okay I revert it (so the secret treasures will never be seen :)